src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -350,7 +349,11 @@ class boss_vazruden_the_herald : public CreatureScript
                     if (Creature* Vazruden = me->SummonCreature(NPC_VAZRUDEN, VazrudenMiddle[0], VazrudenMiddle[1], VazrudenMiddle[2], 0, TEMPSUMMON_CORPSE_TIMED_DESPAWN, 6000000))
                         VazrudenGUID = Vazruden->GetGUID();
                     if (Creature* Nazan = me->SummonCreature(NPC_NAZAN, VazrudenMiddle[0], VazrudenMiddle[1], VazrudenMiddle[2], 0, TEMPSUMMON_CORPSE_TIMED_DESPAWN, 6000000))
-                        NazanGUID = Nazan->GetGUID();
+					{
+					    NazanGUID = Nazan->GetGUID();
+					    Unit* player = Nazan->SelectNearestPlayer(60.00f);//nazan start attack
```

Comment:
This feels really ugly. You already have a threat list of players engaged with you - why not just use that?

Reply:
thank you for your advise,  I just come here for few days,  I found trouble, and I do it on my own way,  your advise is very precious.  Trust me , I'm using google translate to type English.  I'll fix it next day,  I must go to bed , goodnight.